packages/discovery-styles/scss/components/document-preview/_document-preview.scss

```diff
@@ -1,3 +1,4 @@
+@import '~carbon-components/scss/components/loading/loading';
```

Comment:
We have `discovery-styles/scss/carbon-styles.scss` to refer Carbon styles. I think it's good to move this to the file.